subprojects/ce-login/cli/CliCeLoginV2.cpp

```diff
@@ -44,9 +44,8 @@ CeLoginRc CeLogin::createCeLoginAcfV2Payload(
     std::string sPasswordHashHexString;
     std::string sSaltHexString;
     std::string sReplayId = cli::generateReplayId();
-    const bool sIsAdminReset = ("adminreset" == sArgsV2.mType);
-    const bool sIsResourceDump = ("resourcedump" == sArgsV2.mType);
-    const bool sIsBmcShell = ("bmcshell" == sArgsV2.mType);
+
+    const AcfType sAcfType = CeLogin::getAcfTypeFromString(sArgsV2.mType);
```

Comment:
I don't see anywhere that you check for getting back an invalid type from this function. If the type was invalid it looks like we might just return success.